cypress/integration/articleFlows/postSidebarActions.spec.js

```diff
@@ -46,20 +43,20 @@ describe('Post sidebar actions', () => {
     cy.get('@dropdownButton').click();
 
     // Check dropdown is closed by asserting the first option isn't visible
-    cy.findByRole('img', {
-      name: /^Copy article link to the clipboard$/i,
-    }).should('not.exist');
+    cy.get('#copy-post-url-button').should('not.be.visible');
```

Comment:
```suggestion
    cy.get('@copyPostUrlButton').should('not.exist');
```

Just one final `get` we could change to use the alias you created 👍